### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by dev-enforced**
In this pull request one can find the CSS for the text utilities provided by this component library.Variations can be found in terms of:
1) Font Size;
2) Font Weights;
3) Colors;
4) Headings;
5) Text Positions (right,left and center)

Here is the live preview link for the files being changed: https://623a07726dc1730008106390--gentle-ui.netlify.app/documentation/text-utilities.html

### CODE REVIEW

File: documentation/text-utilities.html [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Exo:wght@300;500;600;700&display=swap" />
+    <link rel="stylesheet" href="../gentle-imports.css" />
+    <link rel="stylesheet" href="documentation.css" />
+    <link rel="icon" href="https://img.icons8.com/stickers/100/000000/g.png " />
+    <title>Text Utilities | Gentle UI</title>
+</head>
+
+<body>
+    <div class="documentation-layout">
+        <div class="navbar p-4">
```
Comment (adith-a-danthi) [inline documentation/text-utilities.html:21]: You can use `nav` tag here

File: documentation/text-utilities.html [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                    <a href="../index.html" class="link-none">HOME</a>
+                </li>
+                <li>
+                    <a href="https://github.com/dev-enforced/Gentle_UI" class="link-none">
+                        <i class="fab fa-github fa-2x"></i>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <div class="sidebar p-6">
```
Comment (adith-a-danthi) [inline documentation/text-utilities.html:38]: You can use `aside` tag for sidebar

File: documentation/text-utilities.html [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+                    <h6><a class="link-none" href="navigation.html">NAVIGATION</a></h6>
+                </li>
+                <li class="p-2 my-2">
+                    <h6>
+                        <a class="link-none" href="text-utilities.html">TEXT UTILITIES</a>
+                    </h6>
+                </li>
+            </ul>
+        </div>
+        <div class="main-content p-5">
```
Comment (adith-a-danthi) [inline documentation/text-utilities.html:78]: You can use a `main` tag here

**Review by adith-a-danthi — COMMENTED** [review]
Looks Good. Clean code. Just a few suggestions
- Try to use semantic tags where possible.
- (Optional) - You could have used `text-sm/md/lg` classes for the regular text and `heading-1/2/3` classes for the headings

**Review by ShreyashMahajan — APPROVED** [review]
good to go

File: documentation/text-utilities.html (hunk 3 above)
Comment (dev-enforced) [reply documentation/text-utilities.html:78]: Yes will use keep in mind while making the upcoming pages of the project

File: documentation/text-utilities.html (hunk 2 above)
Comment (dev-enforced) [reply documentation/text-utilities.html:38]: Yes will keep in mind while making the upcoming pages of the project

**Review by AshwiniSetty — APPROVED** [review]
Good to merge! Code looks clean

**Pull request merged**